dll/ntdll/ldr/ldrapi.c

```diff
@@ -426,7 +427,7 @@ LdrLoadDll(IN PWSTR SearchPath OPTIONAL,
         LdrpTopLevelDllBeingLoaded = OldTldDll;
 
         /* Release the lock */
-        LdrUnlockLoaderLock(LDR_LOCK_LOADER_LOCK_FLAG_RAISE_ON_ERRORS, Cookie);
+        LdrUnlockLoaderLock(LDR_UNLOCK_LOADER_LOCK_FLAG_RAISE_ON_ERRORS, Cookie);
```

Comment:
Regarding these typo fixes, note that
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/kernel32/client/loader.c;hb=3fb5957de14da885a16324fca028d9f0ea7fdf37#l659
also has this typo (this is the other single one remaining).

Reply:
Committed.